api-processor/src/main/java/com/synopsys/integration/alert/processor/api/extract/model/ProcessedProviderMessage.java

```diff
@@ -55,4 +59,13 @@ public T getProviderMessage() {
         throw new IllegalStateException("Combining models had more than two results");
     }
 
+    @Override
+    public int hashCode() {
+        return HashCodeBuilder.reflectionHashCode(17, 37, this, false, Stringable.class, "notificationIds");
```

Comment:
Why did you choose to use this constructor? Elsewhere in the code we use a much simpler version:
`HashCodeBuilder.reflectionHashCode(this, EXCLUDED_COMPARISON_FIELDS);`

Reply:
Implemented the simpler version appears to work as expected.  The notable difference is Stringable i used but that class extends Object and we override the equals and hashcode methods in ProcessedProviderMessage.  The simpler version should work correctly.